src/main/java/org/benetech/servicenet/service/impl/ClientProfileServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package org.benetech.servicenet.service.impl;
```

Comment:
Shouldn't we also save clients in the auth service?

Reply:
I believe this resource should act as a proxy - please have a look at the user profiles